template/prettier.config.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2020, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module.exports = require('@sumup/foundry/prettier')(
+  {},
+  {
+    quoteProps: 'consistent',
+    proseWrap: 'always',
```

Comment:
Again, I'd prefer to change this in Foundry to keep it consistent across projects.

Reply:
If we use the default Foundry configs, we can also remove the config files from the template and instead run `yarn foundry init` as a `postinstall` script. This would allow the users to pick their own setup.

Reply:
Oh yes, brilliant idea. I only realized that `postinstall` works when trying to fix a problem with the nested ESLint config.

Reply:
That's what inspired me 😄

Reply:
Done